### PR TITLE
limit generated messages to prevent M_TOO_LARGE

### LINKED ISSUE #8: listusers results in M_TOO_LARGE
My slack instance has ~800 users and listusers fails with the following

```
Sep-24 10:28:10.408 [bot-sdk-MatrixLiteClient] error: (REQ-6) { errcode: 'M_TOO_LARGE', error: 'event too large' }
```

The bot messages should probably be split up in multiple events once it exceeds the event size limit.

### PR DESCRIPTION
**Pull request by twouters**
Just a proposal to fix #8 

I didn't want to limit all messages because it's:
- too hard to properly split a string without breaking markdown or html elements
- probably impossible to keep track of puppeted event id's

Instead I've decided that it's probably better to only limit the size of self-generated messages
It's just an idea. If there's a better way to do this, feel free to comment.

### CODE REVIEW

**Comment by twouters** [comment]
Oh, right, `maxMsgSize` is just sort of an arbitrary number (it's what Slack defines as maximum size for messages in input field)

File: src/botprovisioner.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export class BotProvisioner {
 		}
 		const roomId = event.room_id;
 		const sender = event.sender;
+		const maxMsgSize = 4000;
```
Comment (Sorunome) [inline src/botprovisioner.ts]: can you maybe make this a global constant on top in all caps? Like, outside the class `const MAX_MSG_SIZE = 4000;`

**Review by Sorunome — CHANGES_REQUESTED** [review]
Looks good, just one small nitpicking!

**Review by Sorunome — APPROVED** [review]

**Comment by Sorunome** [comment]
Thanks a lot!

**Pull request merged**